src/index.js

```diff
@@ -1 +1,38 @@
-import './style.css';
\ No newline at end of file
+import './style.css';
+import Game from './game';
+
+const refresh = document.querySelector('.refresh');
+const nameD = document.querySelector('.name');
+const scoreD = document.querySelector('.score');
+const submit = document.querySelector('.submit');
+const list = document.querySelector('.list');
+
+const gameDom = (namev, scorev) => {
+  const li = document.createElement('li');
+  const name = document.createElement('h4');
+  const score = document.createElement('h4');
+  name.innerHTML = namev;
+  score.innerHTML = scorev;
+  li.appendChild(name);
+  li.appendChild(score);
+  list.appendChild(li);
+};
+refresh.addEventListener('click', () => {
+  list.innerHTML = '';
+  Game.refresh();
+});
```

Comment:
- [ ] It would be better if when you click on the refresh page the data doesn't disappear. The job of the refresh button is to receiving data from the API and parsing the JSON. kindly fix this minor issue. 

![Capture](https://user-images.githubusercontent.com/64173189/136498363-980938ef-7a09-4da9-be11-a9e1a4b4ad00.JPG)

🛠️ 😄

Reply:
Wow, thanks will fix that immediately